Dockerfile

```diff
@@ -76,5 +86,4 @@ RUN mkdir -p \
   && brew cleanup \
   && { git -C .linuxbrew/Homebrew config --unset gc.auto; true; } \
   && { git -C .linuxbrew/Homebrew config --unset homebrew.devcmdrun; true; } \
-  && rm -rf .cache \
```

Comment:
Why remove this?

Reply:
because we're using a cache volume now here: https://github.com/Homebrew/brew/pull/17584/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R70. 

Cache volume are not part of the final image so no need to remove anything.

Reply:
Ah, yeh, I see now, thanks.